job-dsls/jobs/srcclr_scan_seed.groovy

```diff
@@ -0,0 +1,88 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def repoList = [
+        'lienzo-core',
+        'drools'
+]
+
+for (repo in repoList) {
+
+    def jobName = 'srcclr-scan-' + "${repo}"
+
+    job(jobName) {
+
+        description('Job responsible for SourceClear verification of' + "${repo}")
+
+        parameters {
+            choiceParam('SCAN_TYPE', ['scm', 'binary'])
+            stringParam('URL','')
+            stringParam('VERSION', '')
+            stringParam('PACKAGE','')
+            stringParam('NAME', '')
+            stringParam('MVNPARAMS', '')
+            choiceParam('PROCESSOR_TYPE', ['cve', 'cvss'])
+            booleanParam('RECURSIVE', false)
+            booleanParam('DEBUGGING', false)
+            booleanParam('TRACING', false)
+            stringParam('SCMVERSION', '')
+            stringParam('THRESHOLD', '1','Threshold from 1 to 10 for cvss processor')
+        }
+
+
+        environmentVariables{
+            groovy('''
+          def map = [:]
+               
+          map.put("RECURSE", Boolean.valueOf("${RECURSIVE}") ? "--recursive" : "")
+          
+          map.put("DEBUG", Boolean.valueOf("${DEBUGGING}") ? "-d" : "");
+          
+          map.put("TRACE", Boolean.valueOf("${TRACING}") ? "--trace" : "")
+          
+          map.put("MVNPARAMETER", "${MVNPARAMS}" !="" ? "--maven-param=${MVNPARAMS}":"")
+          
+          map.put("SCMVERSIONPARAM", "${SCAN_TYPE}" == "scm" ? " --ref=${SCMVERSION}":"")
```

Comment:
spaces around `:` in `" --ref=${SCMVERSION}":""`. It can cause troubles (not sure)

Reply:
See above, please.